apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2252,17 +2369,32 @@ static void libc_stdio_rawinstream_tc(void)
 static void libc_stdio_rawoutstream_tc(void)
 {
 	int fd;
-	char *filename = VFS_FILE_PATH;
 
 	struct lib_rawoutstream_s rawoutstream;
 
-	fd = open(filename, O_RDONLY);
+	/* Negative case, no character is put to the rawoutstream */
```

Comment:
description is wrong. this is failed because fd is opened with read permission and put function tried to write something.